presidio-analyzer/tests/test_spacy_recognizer.py

```diff
@@ -23,6 +27,8 @@ class TestSpacyRecognizer(TestCase):
     #     assert len(results) == 1
     #     assert_result(results[0], entity[0], NER_STRENGTH)
 
+    @pytest.mark.skipif(not nlp_is_available,
```

Comment:
How do we verify that an installation is successful? Usually unit tests fail if something is missing, however in this case we would skip it completely.

Reply:
The idea behind this is that while using spacy and stanza, you only need one or the other.  But I should make the spacy tests required, especially since it makes more sense to use `spacy-stanza`.

However, I do think that requiring a large spacy model for testing is a fundamental design flaw in the tests.  It's mixing the performance/accuracy of the spacy models with the testing of the functionality of the library.  The best solution would be to mock spacy and just create the entities you want for each test ([SO answer from a spacy developer on this topic](https://stackoverflow.com/a/56733926)).  The second best solution would be to have all the tests pass with a minimal spacy model (i.e. `en`).  Then separately have scripts to benchmark how well you want the whole library to function when using the best models.  

No hard feelings if you disagree with me here.  I do understand the idea that having the best model out of the box can create a more desirable experience for users.  (see note below about my internet situation as to why I'm against making the large spacy model a requirement)

Reply:
I completely agree that a mock (or at least a small model) should be the one used for testing. I'll discuss this with the team as there are some implications (for example, we have test that verifies that the code on the demo website is analyzed correctly. This wouldn't work with a mock, but could be turned into an integration test). There are some other implications on the way we do CI. @balteravishay what are you thoughts on this?

Reply:
I actually changed this to default to the big model but now you can use the small model with `pytest --runfast`.

Reply:
Cool, thanks. 
Are you considering to implement the spacy-stanza solution?